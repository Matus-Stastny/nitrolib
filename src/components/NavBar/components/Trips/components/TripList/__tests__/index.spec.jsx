// @flow strict
import * as React from "react";
import { shallow } from "enzyme";

import { TripListUnwrapped as TripList } from "..";

const list: any = {
  edges: [
    {
      node: {
        __id: "id",
        __typename: "typename",
        destinationImageUrl: "url",
      },
    },
  ],
};

const lang = "en";

describe("#TripList", () => {
  test("render", () => {
    const wrapper = shallow(<TripList list={list} lang={lang} />);

    expect(wrapper).toMatchSnapshot();
  });
});
