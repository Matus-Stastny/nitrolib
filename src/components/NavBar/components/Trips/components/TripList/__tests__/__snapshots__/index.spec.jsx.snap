// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#TripList render 1`] = `<Fragment />`;
